### PR TITLE
Add more top tier conferences and papers

### PR DESCRIPTION
**Pull request by bdqnghi**
I would suggest adding 2 more top tier conferences in software engineering:
- ICSE: https://www.icse2018.org/
- FSE: https://conf.researchr.org/home/fse-2018

And 2 papers published in ICSE this year and FSE last year about source code modeling, one of them is mine :) 

ICSE and FSE are the 2 top tier conferences in software engineering (A* ranking). Recently, a lot of papers about source code modeling by using deep learning are published in these conferences.

Here's the short overview of the 2 papers:
1) Hierarchical Learning of Cross-Language Mappings through Distributed Vector Representations for Code: learning the distributed vector representation of code in a cross-languages setting, the results show that it can mine a lot of API mappings and outperform the baselines)
2) Are deep learning the best for source code modeling?: an empirical study to compare between neural-based method and n-gram based method for source code generation

Signed-off-by: Nghi Bui [bdqnghi@gmail.com](url)

### CODE REVIEW

**Review by vmarkovtsev — APPROVED** [review]

**Comment by vmarkovtsev** [comment]
Thank you so much for your contribution. You seem to miss a new line between the commit message title and the "signed-off-by" note. Could you please amend the commit and I will merge it at once!

File: README.md
Comment (EgorBu) [inline README.md]: `"badges/4-pages-gray.svg" alt="12-pages"` - it should be 4 pages both

**Review by EgorBu — CHANGES_REQUESTED** [review]
Thank you for the contribution! A couple of things before merging:
1) Please provide in PR comment with a short review of each article and conference to show why it should be included.
2) Minor fix in PR.

**Comment by vmarkovtsev** [comment]
@egorbu (1) is already done. We cannot ask for a longer review because ourselves have not done it :-)

**Comment by vmarkovtsev** [comment]
@bdqnghi can you please fix the DCO

**Comment by bdqnghi** [comment]
Hi, i have updated the missing parts, can you check again?

**Comment by vmarkovtsev** [comment]
All good, the signed-off-by is missing in the new commits and is still corrupted in the first one though.

**Comment by bdqnghi** [comment]
ok, do i need to do something to fix it? or you merge it by yourself?

**Comment by bdqnghi** [comment]
ok since I made a lot of mistakes on this PR, i will close it and create a new one

**Pull request closed without merging**
